cli/dcoscli/service/main.py

```diff
@@ -123,3 +146,170 @@ def _shutdown(service_id):
 
     mesos.DCOSClient().shutdown_framework(service_id)
     return 0
+
+
+def _log(follow, lines, config_file, service, file_):
+    """Prints the contents of the logs for a given service.  The service
+    task is located by first identifying the marathon app running a
+    framework named `service`.
+
+    :param follow: same as unix tail's -f
+    :type follow: bool
+    :param lines: number of lines to print
+    :type lines: int
+    :param config_file: SSH config file.  Used for marathon.
```

Comment:
Can we add that to the comment? Something along the lines of `SSH config file; used when service is set to "marathon".`